examples/cg/cg.py

```diff
@@ -32,48 +32,41 @@ def fit(A, b, x0, tol, max_iter):
         b = xp.dot(r1.T, r1) / xp.dot(r0.T, r0)
         p = r1 + b * p
         r0 = r1
-    msg = 'Failed to converge. Increase max-iter or tol.'
-    print(msg)
+    print('Failed to converge. Increase max-iter or tol.')
     return x
 
 
 def run(gpu_id, tol, max_iter):
     '''CuPy Congugate gradient example
 
     Solve simultaneous linear equations, Ax = b.
-    'x' is computed in two ways. To check whether the answers are correct,
-    'b' is computed from matrix multiplication of 'A' and 'x' in each case,
-    and printed.
+    'A' and 'x' are created randomly and 'b' is computed by 'Ax' at first.
+    'x' is the answer and computed in two ways. To check whether the computed
```

Comment:
Could you consider the suggestion of wording?

"Then, 'x' is computed from 'A' and 'b' in two ways, namely with CPU and GPU. To evaluate the accuracy of computation, the Euclidean distances between the answer 'x' and the reconstructed 'x' are computed."